src/bin/pg_autoctl/primary_standby.c

```diff
@@ -64,7 +64,7 @@ GUC postgres_default_settings[] = {
 
 GUC citus_default_settings[] = {
 	DEFAULT_GUC_SETTINGS_FOR_PG_AUTO_FAILOVER,
-	{ "shared_preload_libraries", "citus" },
+	{ "shared_preload_libraries", "'citus'" },
```

Comment:
I did not get why we need this

Reply:
I saw the configuration file in the output and it didn't have any quotes around the shared library name and I though that was not right...